corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_form_setting.html

```diff
@@ -74,6 +74,20 @@
         </div>
       {% endif %}
     </div>
+    {% if request|toggle_enabled:'FOLLOWUP_FORMS_AS_CASE_LIST_FORM' %}
+      <div class="form-group" id="case_list_form-expression" data-bind="visible: caseListForm()">
+        <label class="{% css_label_class %} control-label">
+          {% trans "XPath Expression to Enable Case List Form" %}
```

Comment:
Nitpick: maybe call this "Display Condition"? HQ uses that phrase in a bunch of places.